### PR TITLE
Move address input initialize to function so it can be called if new …

### PR DESCRIPTION
**Pull request by mattijevi**
…elements are added

I used django address with bootstrap modals that were loaded after the initial page load. 

If you call the function `django_input_address_setup()` like this:

```
$(your modal).on('shown.bs.modal', function(event) {
    django_input_address_setup();
});
```

The new input should get the address. 
address/css/address.css also needs to be included for z-index

### CODE REVIEW

File: address/static/address/css/address.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.pac-container {
+    z-index: 1051 !important;
```
Comment (furious-luke) [inline address/static/address/css/address.css]: Is this directly relevant to `django-address`? It seems like this is something that should belong in a CSS file specific to your site.

**Comment by furious-luke** [comment]
Thanks for the pull-request! I think this kind of thing may be better written as a jQuery plugin as opposed to a function, so as to make it more easily accessible.

**Comment by mattijevi** [comment]
That's a good idea. I could rewrite it if you like

**Comment by mattijevi** [comment]
The css can be removed. It is only to make the input work on modals.

I created this:
https://github.com/mattijevi/django-address/blob/master/address/static/address/js/address.js

**Comment by LifeMoroz** [comment]
@mattijevi can you resolve conflicts ?

**Comment by banagale** [comment]
This changes a pretty important little script and I'm hesitant to take on review and testing of it right now.  @LifeMoroz Are you still waiting on this?

**Comment by banagale** [comment]
Closing as stale
